ckb-sync-mainnet/script/sync-mainnet.sh

```diff
@@ -214,8 +215,8 @@ function main() {
             terraform_apply
             rust_build
             ansible_deploy_ckb
-            ansible_wait_ckb_synchronization
-            github_add_comment "$(markdown_report)"
+            #ansible_wait_ckb_synchronization
```

Comment:
Why comment out these lines?

Reply:
liya suggest to comment/disable actual sync operation when test the patch will work...guess she may mean to do test on my own ckb repo instead of this ckb repo.
It's really mess when I really want to do test in Github actions(Sync Mainnet), I do need help on this...

Reply:
I now revert this 2 lines.